docker-files/drs-server/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6-alpine
```

Comment:
TODO: update/test newer python (low priority)

Reply:
I attempted to update to newer versions, but many resulted in build issues. The newest I could find that worked was python:3.7-alpine. So the DRS image is now built from that image as of 
19ddec1.